### PR TITLE
i18n(es): Translated `svelte.mdx`

### PR DESCRIPTION
**Pull request by Waxer59**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.
Translated `svelte.mdx` to spanish

### CODE REVIEW

File: src/content/docs/es/guides/integrations-guide/svelte.mdx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+**`svelte.config.js`**
+
+```js
+export default {
+  preprocess: [],
+};
+```
+
+## Intellisense for TypeScript
```
Comment (dreyfus92) [inline src/content/docs/es/guides/integrations-guide/svelte.mdx:130]:
```suggestion
## Intellisense para TypeScript
```

File: src/content/docs/es/guides/integrations-guide/svelte.mdx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+
+```js
+import { vitePreprocess } from '@astrojs/svelte';
+
+export default {
+  preprocess: vitePreprocess(),
+};
+```
+
+Este archivo de configuración se agregará automáticamente por ti cuando ejecutes `astro add svelte`.
```
Comment (dreyfus92) [inline src/content/docs/es/guides/integrations-guide/svelte.mdx:146]:
```suggestion
Este archivo de configuración se agregará automáticamente cuando ejecutes `astro add svelte`.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Te dejo unas sugerencias @Waxer59, buen trabajo 😄

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 💜

**Pull request merged**